airbyte-integrations/connectors/source-coingecko-coins/source_coingecko_coins/manifest.yaml

```diff
@@ -64,4 +64,4 @@ streams:
 check:
   stream_names:
     - "market_chart"
-    - "history"
+    - "history"
```

Comment:
Please remove the file from this PR as it is unrelated to the LinkedIn Ads source connector.